HabitTracker/HabitTrackerLibrary/HabitDatabase.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Microsoft.Data.Sqlite;
+
+namespace HabitTrackerLibrary;
+
+public static class HabitDatabase
+{
+    private static readonly string _connectionString = "DataSource=HabitTracker.db";
+    public static Dictionary<string, Habit.Amount?> HabitTypes = new();
+
+    public static void CreateDatabase()
+    {
+        using (var connection = new SqliteConnection(_connectionString))
```

Comment:
Remember that you can use the simpler "Using Declaration" syntax to save an indentation level against the "Using Statement" syntax:

❌ 
```
// Using Statement
using (var connection = new SqliteConnection(connectionString))
{
    // ...
}
```

✔️ 
```
// Using Declaration
using var connection = new SqliteConnection(connectionString);
// ...
```